sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java

```diff
@@ -217,41 +162,15 @@ public void catchHttpErrorOnRequestAsync() {
         }
     }
 
-    public void failedMessagesAsync() {
-        SmsAsyncClient smsClient = createSmsAsyncClientUsingAzureKeyCredential();
-
-        SmsSendOptions options = new SmsSendOptions();
-        options.setDeliveryReportEnabled(true);
-        options.setTag("Tag");
-
-        Mono<Response<Iterable<SmsSendResult>>> sendResults = smsClient.sendWithResponse(
-            "<from-phone-number>",
-            Arrays.asList("<to-phone-number1>", "<to-phone-number2>"),
-            "Hi",
-            options /* Optional */);
-
-        Mono <Iterable<SmsSendResult>> resultOfEachMessage = sendResults.flatMap(response -> {
-            Iterable<SmsSendResult> iterableResults = response.getValue();
-            for (SmsSendResult result : iterableResults) {
-                if (!result.isSuccessful()) {
-                    System.out.println("Successfully sent this message: " + result.getMessageId() + " to " + result.getTo());
-                } else {
-                    System.out.println("Something went wrong when trying to send this message " + result.getMessageId() + " to " + result.getTo());
-                }
-            }
-            return Mono.just(iterableResults);
-        });
-
-    }
-
-    public void failedMessagesSync() {
+    public void failedMessages() {
```

Comment:
hmm can we have a more descriptive name?
SendMessageTroubleShooting is the .NET name
maybe sendMessageAndCheckForFailure ?